java/Oleksii-Riabov/src/com/zoolatech/lecture1/tasks/_1/Task1.java

```diff
@@ -0,0 +1,26 @@
+package com.zoolatech.lecture1.tasks._1;
```

Comment:
please remove files that belong to a first and second lectures from this PR - you've already created a one with these homeworks, this might result into a merge conflict

Reply:
How would you recommend to do this? The best option that i can see is to use git rebase -i <sha1-commit-hash> command and just "drop" redundant files from current branch.

Reply:
Solved.